frontend/vuejs/src/views/Reports/ListReport.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+  <div class="main-content">
+    <base-header type="gradient-success" class="pb-6 pb-8 pt-5 pt-md-8"></base-header>
+    <div class="content">
+      <h3 class="title">{{ $t("report_screen.label.title") }}</h3>
+      <ul class="nav nav-pills switch-btn">
+        <li class="nav-item" @click="switchTable(true)">
+          <button class="btn nav-link" v-bind:class="{ active: status }">{{ $t("report_screen.label.done") }}</button>
```

Comment:
gom 2 class lại và sử dụng shorthand cho gọn nhé
https://vuejs.org/v2/guide/class-and-style.html